client/my-sites/site-settings/section-import.jsx

```diff
@@ -105,7 +105,7 @@ class SiteSettingsImport extends Component {
 			{
 				args: { title },
 				components: {
-					a: <a href="https://support.wordpress.com/import/" />,
+					a: <a href="https://en.support.wordpress.com/import/" />,
```

Comment:
Good question. There's no consistency in relation to the support links unfortunately. We've peppered both `en.support.wordpress.com/...` and `support.wordpress.com/...` about the app, and for some links, will modify them for `es`, `de` and `pr-br`.

In short, my preference is also to drop `en`. It's superfluous since we'll redirect to `en` anyway. Moreover, if we choose to perform any extra redirect magic based on locale-guessing later we can do so without worrying about extra noise. 

There's a concerted effort at the moment to improve the support docs across our major languages, so without doubt we'll be revisiting all these links in the near future.